python/ray/serve/_private/deployment_state.py

```diff
@@ -2451,7 +2473,13 @@ def update(self) -> bool:
                 deployment_state.autoscale(
                     current_num_ongoing_requests, current_handle_queued_queries
                 )
-            deleted, recovering = deployment_state.update()
+
+            deleted, recovering, upscale, downscale = deployment_state.update()
+            if upscale:
+                upscales[deployment_name] = upscale
+            if downscale:
+                downscales[deployment_name] = downscale
```

Comment:
```suggestion
            deleted, recovering, upscales[deployment_name], downscales[deployment_name] = deployment_state.update()
```

Reply:
I don't want to add to upscales/downscales dict if the deployment has no upscale or downscale request.